ledger/src/blockstore_db.rs

```diff
@@ -1948,36 +1948,9 @@ std::env::var("SOLANA_METRICS_ROCKSDB_PERF_SAMPLES_IN_1K")
 
 }
 
-impl<C> LedgerColumn<C>
-where
-    C: TypedColumn + ColumnName + ColumnMetrics,
-{
-    pub fn get(&self, key: C::Index) -> Result<Option<C::Type>> {
-        let mut result = Ok(None);
-        let is_perf_context_enabled = Self::maybe_collect_perf_context();
-        if let Some(serialized_value) = self.backend.get_cf(self.handle(), &C::key(key))? {
-            let value = deserialize(&serialized_value)?;
-
-            result = Ok(Some(value))
-        }
-
-        if is_perf_context_enabled {
-            self.report_read_perf_context(C::rocksdb_get_perf_metric_header(&self.column_options));
-        }
-        result
-    }
-
-    pub fn put(&self, key: C::Index, value: &C::Type) -> Result<()> {
-        let serialized_value = serialize(value)?;
-
-        self.backend
-            .put_cf(self.handle(), &C::key(key), &serialized_value)
-    }
-
-    pub fn delete(&self, key: C::Index) -> Result<()> {
-        self.backend.delete_cf(self.handle(), &C::key(key))
-    }
+struct RocksMetricsUtils {}
```

Comment:
nit: Given that there is no data, this struct appears to be solely for grouping. As such, I think it may be preferable to use `mod` instead of `struct`.